tests/e2e/test-snippets/test-snippets.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Plugin name: E2E test snippets
```

Comment:
WDYT about naming it as "GLA Test Snippets" to be consistent with the "Gla Test Data" name ?

Reply:
hm.. GLA is quite redundant, as we're in GLA repo, same goes for E2E ;) as that `e2e/`folder, but I' all for consistency :)